testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -6680,3 +6688,1393 @@ fn test_flash_block_skip_tenure() {
+    let mut btc_regtest_controller = BitcoinRegtestController::with_burnchain(
+        conf.clone(),
+        None,
+        Some(burnchain_config.clone()),
+        Some(btc_should_keep_running.clone()),
+    );
+    let http_origin = format!("http://{}", &conf.node.rpc_bind);
+
+    btc_regtest_controller.bootstrap_chain(201);
+
+    eprintln!("Chain bootstrapped...");
+
+    let mut run_loop = neon::RunLoop::new(conf.clone());
+    let blocks_processed = run_loop.get_blocks_processed_arc();
+
+    let client = reqwest::blocking::Client::new();
+    let channel = run_loop.get_coordinator_channel().unwrap();
+
+    thread::spawn(move || {
+        run_loop.start(Some(burnchain_config), 0);
+        btc_should_keep_running.store(false, Ordering::SeqCst);
+    });
+
+    // give the run loop some time to start up!
+    wait_for_runloop(&blocks_processed);
+
+    // first block wakes up the run loop
+    next_block_and_wait(&mut btc_regtest_controller, &blocks_processed);
+
+    // first block will hold our VRF registration
+    next_block_and_wait(&mut btc_regtest_controller, &blocks_processed);
+
+    // second block will be the first mined Stacks block
+    next_block_and_wait(&mut btc_regtest_controller, &blocks_processed);
+
+    let sort_height = channel.get_sortitions_processed();
+
+    // let's query the miner's account nonce:
+    let account = get_account(&http_origin, &miner_account);
+    assert_eq!(account.balance, 0);
+    assert_eq!(account.nonce, 1);
+
+    // and our potential spenders:
+    let account = get_account(&http_origin, &spender_addr);
+    assert_eq!(account.balance, total_bal as u128);
+    assert_eq!(account.nonce, 0);
+
+    let account = get_account(&http_origin, &spender_2_addr);
+    assert_eq!(account.balance, total_bal as u128);
+    assert_eq!(account.nonce, 0);
+
+    let account = get_account(&http_origin, &spender_3_addr);
+    assert_eq!(account.balance, total_bal as u128);
+    assert_eq!(account.nonce, 0);
+
+    let pox_info = get_pox_info(&http_origin);
+    assert_eq!(pox_info.next_cycle.stacked_ustx, 0);
+
+    // TODO (hack) instantiate the sortdb in the burnchain
+    let _ = btc_regtest_controller.sortdb_mut();
+
+    for sk in [spender_sk, spender_2_sk, spender_3_sk].iter() {
+        let tx = make_contract_call(
+            sk,
+            0,
+            260,
+            &StacksAddress::from_string("ST000000000000000000002AMW42H").unwrap(),
+            "pox",
+            "stack-stx",
+            &[
+                Value::UInt(half_stacked_bal as u128),
+                execute(&format!(
+                    "{{ hashbytes: 0x{}, version: 0x00 }}",
+                    pox_pubkey_hash
+                ))
+                .unwrap()
+                .unwrap(),
+                Value::UInt(sort_height as u128),
+                Value::UInt(4),
+            ],
+        );
+
+        // okay, let's push that stacking transaction!
```

Comment:
I would delete this comment and comment on what the loop does before the `for` line.